dbbackup/db/base.py

```diff
@@ -0,0 +1,104 @@
+import shlex
+from tempfile import SpooledTemporaryFile
+from subprocess import Popen
+from importlib import import_module
+from dbbackup import settings, utils
+
+CONNECTOR_MAPPING = {
+    'django.db.backends.sqlite3': 'dbbackup.db.sqlite.SqliteConnector',
+    'django.db.backends.mysql': 'dbbackup.db.mysql.MysqlDumpConnector',
+    'django.db.backends.postgresql': 'dbbackup.db.postgres.PgDumpConnector',
+    'django.db.backends.postgresql_psycopg2': 'dbbackup.db.postgres.PgDumpConnector',
+    'django.db.backends.oracle': None,
+    'django_mongodb_engine': 'dbbackup.db.mongo.MongoDumpConnector',
```

Comment:
These mappings should also have the spatial backends listed here:

https://docs.djangoproject.com/en/1.9/ref/contrib/gis/db-api/#module-django.contrib.gis.db.backends

I'm guessing that the database engines' dumping tools won't change. I know in Postgres it doesn't change.